# A custom message for robot position and velocity
float32 pos_x
float32 pos_y
float32 vel_x
float32 vel_z
